examples/using-graphql-typegen/package.json

```
@@ -0,0 +1,34 @@
+{
+  "name": "using-graphql-typegen",
+  "version": "1.0.0",
+  "private": true,
+  "description": "Example on how to use GraphQL Typegen in Gatsby",
+  "author": "LekoArts",
+  "scripts": {
+    "develop": "gatsby develop",
+    "start": "gatsby develop",
+    "build": "gatsby build",
+    "serve": "gatsby serve",
+    "clean": "gatsby clean",
+    "typecheck": "tsc --noEmit",
+    "lint": "eslint . --ignore-path .gitignore --ext .ts,.tsx",
```

Since you already specified `overrides` section in your ESLint config, you no need to specify extensions
```suggestion
    "lint": "eslint --ignore-path .gitignore .",
```